src/ImmutableX.ts

```diff
@@ -350,6 +356,87 @@ export class ImmutableX {
       });
   }
 
+  /**
+   * Get a list of metadata refreshes
+   * @param ethSigner - the L1 signer
+   * @param collectionAddress - the Collection contract address
+   * @param pageSize - the page size of the result
+   * @param cursor - the cursor
+   * @returns a promise that resolves with the requested metadata refreshes
+   * @throws IMXError
+   */
+  public listMetadataRefreshes(
+    ethSigner: EthSigner,
+    collectionAddress?: string,
+    pageSize?: number,
+    cursor?: string,
+  ) {
+    return this.workflows
+      .listMetadataRefreshes(ethSigner, collectionAddress, pageSize, cursor)
+      .then(res => res.data)
+      .catch(err => {
+        throw formatError(err);
+      });
+  }
+
+  /**
+   * Get a list of metadata refresh errors
+   * @param ethSigner - the L1 signer
+   * @param refreshId - the metadata refresh ID
+   * @param pageSize - the page size of the result
+   * @param cursor - the cursor
+   * @returns a promise that resolves with the requested metadata refresh errors
+   * @throws IMXError
+   */
+  public getMetadataRefreshErrors(
```

Comment:
Should this be `listMetadataRefreshErrors` to be consistent with other collection-based APIs?

Reply:
I believe this is  named `get` because it gets data for a single `refreshId`